r-package/grf/vignettes/categorical_inputs.Rmd

```diff
@@ -0,0 +1,78 @@
+---
+title: "Categorical inputs"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{ci_and_num.trees}
```

Comment:
New name!

And remember you have to add an entry here:

https://github.com/grf-labs/grf/blob/master/r-package/grf/_pkgdown.yml#L10

Reply:
Thanks!